### PR TITLE
Important! Template update for nf-core/tools v2.14.0

### PR DESCRIPTION
**Pull request by mirpedrol**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nf-test test tests/ --verbose --profile +docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -160,104 +160,102 @@ try {
...
-        shifter.enabled        = false
-        singularity.enabled    = false
+        conda.enabled           = true
+        docker.enabled          = false
+        singularity.enabled     = false
+        podman.enabled          = false
+        shifter.enabled         = false
+        charliecloud.enabled    = false
+        conda.channels          = ['conda-forge', 'bioconda', 'defaults']
+        apptainer.enabled       = false
```
Comment (maxulysse) [inline nextflow.config:176]: am I the only one that likes ordering items?

**Comment by maxulysse** [comment]
Looking good

**Comment by mirpedrol** [comment]
Linting tests are due to using the current version 2.13.1 of nf-core/tools, they pass when using the dev version 😃 👍

File: assets/multiqc_config.yml [hunk 2]
```diff
@@ -3,9 +3,11 @@ custom_logo_url: https://github.com/nf-core/sarek/
 custom_logo_title: "nf-core/sarek"
 
 report_comment: >
+
```
Comment (maxulysse) [inline assets/multiqc_config.yml:6]:
```suggestion
```

File: assets/multiqc_config.yml [hunk 3]
```diff
@@ -3,9 +3,11 @@ custom_logo_url: https://github.com/nf-core/sarek/
...
   This report has been generated by the <a href="https://github.com/nf-core/sarek/tree/dev" target="_blank">nf-core/sarek</a>
   analysis pipeline. For information about how to interpret these results, please see the
   <a href="https://nf-co.re/sarek/dev/docs/output" target="_blank">documentation</a>.
+
```
Comment (maxulysse) [inline assets/multiqc_config.yml:10]:
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/sare
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
 - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
 - [ ] Make sure your code lints (`nf-core lint`).
-- [ ] Ensure the test suite passes (`nf-test test tests/ --verbose --profile +docker`).
+- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
```
Comment (maxulysse) [inline .github/PULL_REQUEST_TEMPLATE.md:21]:
```suggestion
- [ ] Ensure the test suite passes (`nf-test test tests/ --verbose --profile +docker`).
```

File: conf/test_full.config [hunk 5]
```diff
@@ -15,7 +15,7 @@ params {
     config_profile_description = 'Full test dataset to check pipeline function'
 
     // Input data for full size test
-    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/sarek/testdata/csv/HCC1395_WXS_somatic_full_test.csv'
+    input = params.pipelines_testdata_base_path + 'sarek/testdata/csv/HCC1395_WXS_somatic_full_test.csv'
```
Comment (maxulysse) [inline conf/test_full.config:18]:
```suggestion
    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/sarek/testdata/csv/HCC1395_WXS_somatic_full_test.csv'
```

**Review by maxulysse — DISMISSED** [review]

**Review by maxulysse — DISMISSED** [review]

**Review by maxulysse — APPROVED** [review]

**Review by jfy133 — APPROVED** [review]

**Pull request merged**